src/utils/chain_name_resolver.ts

```diff
@@ -0,0 +1,16 @@
+export enum ChainName {
+    EnergyWebChain = "Energy Web Chain",
+    EnergyWebVoltaTestnet = "Energy Web Volta Testnet",
+    UnknownChain = "Unknown",
+}
+
+export type ChainId = number;
+
+const chains: Map<ChainId, ChainName> = new Map([
```

Comment:
there is chain configs https://github.com/energywebfoundation/iam-client-lib/blob/f254c16febbf9e28963063460bd5d8344739f383/src/iam/chainConfig.ts#L36, which can be used for this

Reply:
+1